source/code/plugins/in_mysql_workload.rb

```diff
@@ -0,0 +1,47 @@
+module Fluent
+  class MysqlWorkloadInput < Fluent::Input
+    Plugin.register_input('mysql_workload', self)
+
+    # Define `router` method to support v0.10.57 or earlier
```

Comment:
omsagent references fluentd 0.12.24. You don't need to support v0.10.57 or earlier.